xrpl/transaction/types/issued_currency.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+type IssuedCurrency struct {
```

Comment:
Isn't this a duplicate of the Asset struct in `amm.go`?

```
type Asset struct {
	Currency string        `json:"currency"`
	Issuer   types.Address `json:"issuer,omitempty"`
}
```

Reply:
Resolved in the comment above